src/view/components/semantic/modules/dropdown/dropdown.ts

```diff
@@ -87,6 +87,10 @@ export class Dropdown implements AfterViewInit, OnDestroy, ControlValueAccessor
     this._modelValue = value;
   }
 
+  focus(){
+    this._$dropdown.children('input.search')[0].focus();
```

Comment:
You could remove the hack that delays the firing of the 'open' event by wrapping this call here in a 
```
if (!$element.exists() && this.attempts < 10){ 
   setTimeout( () => {
      this.attempts++
      this.focus()
    })
if(this.attempts >= 10 ) { // log error
} else {
  this.attempts = 0
  $element[0].focus() // try catch?
}
```

I think it would be much more clear (and much more re-usable) overall.